components/eventing-controller/api/v1alpha1/eventingbackend_types.go

```diff
@@ -27,27 +27,25 @@ type EventingBackendStatus struct {
 	// +optional
 	EventingReady *bool `json:"eventingReady"`
 
+	// Conditions defines the status conditions
 	// +optional
-	SubscriptionControllerReady *bool `json:"subscriptionControllerReady"`
-
-	// +optional
-	PublisherProxyReady *bool `json:"publisherProxyReady"`
```

Comment:
I thought about it, but it would be duplicated information right to have the Ready value both in the conditions and as a boolean? We can discuss this. 

Q: Does fast-integration tests use these individual booleans or just the `eventingReady` boolean?